package.json

```diff
@@ -16,19 +16,19 @@
   "author": "Ultimate Courses",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.4.3",
-    "@babel/preset-env": "^7.4.3",
-    "@babel/register": "^7.4.0",
+    "@babel/core": "^7.7.4",
```

Comment:
These changes might help developers around the globe, installing latest packages.